opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/sort/SortQueryBuilder.java

```diff
@@ -53,11 +58,46 @@ public SortBuilder<?> build(Expression expression, Sort.SortOption option) {
         return SortBuilders.scoreSort().order(sortOrderMap.get(option.getSortOrder()));
       }
       return fieldBuild((ReferenceExpression) expression, option);
+    } else if (expression instanceof FunctionExpression
```

Comment:
Can you create a method somewhere which does this check? It is repeated in mupltiple places in multiple PRs.